scripts/manage/landing_page_secured.md

```diff
@@ -24,7 +24,9 @@ View the logging output of the individual components of your Spinnaker installat
 
 ## Delete Spinnaker
 
-### Generate a script to delete all the resources that were provisioned as part of your Spinnaker installation
+### Generate a cleanup script
+
+This command generates a script that deletes all the resources that were provisioned as part of your Spinnaker installation.
```

Comment:
@dorbin You may want to amend this description

Reply:
Will wait for @dorbin before merging.

Reply:
@duftler 
From the user perspective, does it matter that this "generates" the cleanup script? Do I have to then run the cleanup script? Or does this command do all the work, in which case we should just say this command deletes all the resources, etc.

Reply:
It generates a cleanup script that the user needs to run.

Reply:
Then we'll need another step after running this script, right? Run the resulting cleanup script.
@demobox

Reply:
The script tells the user what to do next. The precise command depends on the environment and the user is instructed what to do next.

@demobox You also need to make these same changes here: https://github.com/GoogleCloudPlatform/spinnaker-for-gcp/blame/master/scripts/manage/landing_page_unsecured.md#L37-L39

Those 2 files need to be kept in sync.

Reply:
I'd like to frame this as running a cleanup script rather than running a generate script that spawns a cleanup script. If possible.

If this script generates another, but includes a prompt/prompts to run the spawned script, can it be considered one cleanup process, regardless of the implementation detail that we're running one script and spawning another?